### PR TITLE
Fix child name validation failures in `testSingleTokenChoiceChildNaming`

### PR DESCRIPTION
**Pull request by ahoppen**
Fixes the naming validation found by the `testSingleTokenChoiceChildNaming` test, which is describes as follows:

Checks standardized naming of children with a single token choice

- If a child has a single keyword as its choice, it should be named `*Keyword` (e.g. `ImportKeyword`)
- If it’s another token kind, name it the same as the the token choice (e.g. `LeftParen`)

I will be opening more PRs like to fix naming inconsistencies over in the next couple of weeks.

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

File: CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift [hunk 1]
```diff
@@ -157,7 +158,7 @@ public let ATTRIBUTE_NODES: [Node] = [
     documentation: "The availability argument for the _specialize attribute",
     children: [
       Child(
-        name: "Label",
+        name: "AvailabilityLabel",
```
Comment (kimdv) [inline CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift:161]: Do we need a `deprecatedName` here`

**Review by kimdv — APPROVED** [review]
🤩

File: CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift (hunk 1 above)
Comment (ahoppen) [reply CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift:161]: Oh, yes. Thanks for catching it.

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Comment by ahoppen** [comment]
@swift-ci Please test macOS

**Review by bnbarham — APPROVED** [review]

**Pull request merged**